index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
```

Comment:
Troubleshoot Prettier. It still should be formatting these files. Right-Click and then click each format document.

Reply:
When looking at the file in VS Code you are using both Tabs and Spaces. Configure the file to use one or the other (preferably spaces).

<img width="149" alt="image" src="https://user-images.githubusercontent.com/12306850/118582686-c0d7a200-b761-11eb-896a-b3592404fff3.png">